DashboardPage.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const DashboardPage = () => {
+  return <h1>Welcome to the dashboard!</h1>;
```

Comment:
The DashboardPage is currently just a static component. Consider adding some dynamic content or functionality.